content/swiftui/concepts/modifiers/modifiers.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: 'Modifiers'
+Description: 'Modifiers are methods that can alter views.'
+Subjects:
+ - 'Mobile Development'
+ - 'Computer Science'
+Tags:
+ - 'SwiftUI'
+ - 'SwiftUI Modifiers'
+CatalogContent:
+ - 'learn-swift'
+ - 'paths/build-ios-apps-with-swiftui'
+---
+
+In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **Modifiers** are methods that can be called on a particular [View](https://www.codecademy.com/resources/docs/swiftui/views).
```

Comment:
```suggestion
In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **modifiers** are methods that can be called on a particular [View](https://www.codecademy.com/resources/docs/swiftui/views).
```

Reply:
I updated this to display the correct term to use here,